parser.go

```diff
@@ -25,7 +25,8 @@ import (
 	"unicode"
 )
 
-var pythonMultiline = regexp.MustCompile("^(\\s+)([^\n]+)")
+const minReaderBufferSize = 64*1024
```

Comment:
I would suggest use 4096 (4KB, which is the size `bufio.NewReader` uses), which reminds default unchanged. 

User should set `ReaderBufferSize` to a larger number when he/she needs to (in your case, set it to 64KB when your program loads the INI file).

Reply:
OK, adopted